src/CMakeLists.txt

```diff
@@ -87,6 +87,8 @@ set( MIOpen_Source
     activ_api.cpp
     api/find2_0_commons.cpp
     argmax_api.cpp
+    graphapi/graphapi.cpp
+    graphapi/graphapi_tensor.cpp
```

Comment:
Please keep this list in the sorted order

Reply:
[resolved]

Reply:
done